### PR TITLE
M10

### PR DESCRIPTION
**Pull request by bytesontherocks**

### CODE REVIEW

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py:4]: Nice - the fact this is such a small change is a great sign that you picked a solid interface design earlier 👍

File: todo_app/data/db_items.py
Comment (JackMeadDev) [inline todo_app/data/db_items.py]: Why do you want a short id? It feels like you're having to write a fair amount of logic to handle this, when we could just rely on the auto-generated id, which will save us having to maintain this in future behaviour

I think we can get rid of all of the references to this, and just use the inbuilt `_id` field to simplify this fairly drastically e.g.:
```
    for it in mdb_items:
        cards.append(Item(str(it['_id']), it['name'], it['status']))
```

This mostly "just works" though there are a couple of places we no longer need to cast the ids to an int. We might then want to not show the ids to a user, but that should be fine

File: todo_app/data/db_items.py
Comment (JackMeadDev) [inline todo_app/data/db_items.py]: Once you're happy with this, don't forget to clear the old code comments

File: todo_app/tests/client_it/test_client.py
Comment (JackMeadDev) [inline todo_app/tests/client_it/test_client.py:36]: Here you seem to be still using the "list" response format from Trello, but that's not what our MongoDB necessarily wants, and I'm not sure you're using the right mocked DB. The easiest solution here is to take advantage of the mocking & the work you've already done to set up your DB:
```python
from todo_app.data.db_items import add_item, move_card_to_new_list

....

def insert_data_to_db():
    add_item('Test card to do')
    done_id = add_item('Test card done')
    move_card_to_new_list(done_id, "To Do", "Done")
```

File: todo_app/data/db_items.py
Comment (JackMeadDev) [inline todo_app/data/db_items.py]: It looks like this function ignores the `current_list_name` variable, so it might make sense to just remove that argument

File: .env.test
Comment (JackMeadDev) [inline .env.test]: I think you're missing an entry for `AZ_MONGODB_DB_NAME` here

**Review by JackMeadDev — APPROVED** [review]
Nice work, I can see the app working happily and you're really close on the tests - I've left a few optional suggestions below but see the comment re the `insert_data_to_db` function for the most important part of fixing the tests (also `.env.test`). Once they're working, very happy for you to merge this, and if I've not been clear enough then don't hesitate to reach out and ask any more questions!

File: todo_app/data/db_items.py
Comment (bytesontherocks) [reply todo_app/data/db_items.py]: yes, absolutely, I didn't know I could use _id directly

**Pull request merged**
